tests/NimblePros.MediatR.Contrib.Test/Behaviors/CachingBehaviorTests.cs

```diff
@@ -0,0 +1,135 @@
+﻿using MediatR;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
+
+namespace NimblePros.MediatR.Contrib.Test.Behaviors;
+
+public class CachingBehaviorTests
+{
+  [Fact]
+  public async Task Should_ReturnSuccess_GivenValidInputWithCacheMiss()
+  {
+    // Arrange
+    var nextMock = new Mock<INext>();
+    var memoryCacheMock = new Mock<IMemoryCache>();
+    var loggerMock = new Mock<ILogger<Mediator>>();
+    var cacheEntryMock = new Mock<ICacheEntry>();
+
+    var cachingBehavior = new CachingBehavior<TestCommand, bool>(memoryCacheMock.Object, loggerMock.Object);
+    var command = new TestCommand() { Name = "Test name" };
+
+    var cacheKey = "NimblePros.MediatR.Contrib.Test.Behaviors.Helpers.TestCommand";
```

Comment:
Use a class level constant e.g. `_testCacheKey`.

Reduce number of magic strings in the class and also keeps tests shorter and thus easier to follow the actual "work" rather than the variable declarations and assignments.